runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1702,6 +1702,29 @@ LABEL(L_virtualDispatchExit)
 
     END_FUNC(_virtualUnresolvedHelper,virUH,7)
 
+ZZ ================================================
+ZZ  PICBuilderRoutine : _jitResolveConstantDynamic
+ZZ ================================================
+    START_FUNC(_jitResolveConstantDynamic,jRCD)
+
+LABEL(_jitResolveConstantDynamicBody)
+    L_GPR r1,eq_cp_inDataSnippet(,r14)
+    LGF_GPR r2,eq_cpindex_inDataSnippet(,r14)
+    L_GPR r3,eq_codeRA_inDataSnippet(,r14)
+    
+LOAD_ADDR_FROM_TOC(rEP,TR_S390jitResolveConstantDynamic)
+    LR_GPR r0,r14
+    BASR r14,rEP 
+    LR_GPR r14,r0
+    L_GPR   r1,eq_literalPoolAddr_inDataSnippet(r14)
+    ST_GPR  r2,0(,r1)
+    L_GPR r2,eq_codeRA_inDataSnippet(,r14)
+    AHI_GPR r2,-6
+    LHI r1,-16380
```

Comment:
Can we use a hex value here for readability? There are some examples in the same file. Might want to also put a comment here saying what we are doing exactly.

Reply:
Let me add comments. I avoided to make sure m4 compiler behaves well
Each line in this file should be contains within certain number letters

Reply:
Yes, generally keep things under 70 characters per line and you should be good. You can add comments on a line preceding the instruction as is the convention throughout the file.